main.py

```diff
@@ -3,14 +3,31 @@
 # throughout this file
 import pygame
 
+from constants import SCREEN_HEIGHT, SCREEN_WIDTH
+
 
 def main():
-    try:
-        pygame.init()
-        print('Starting asteroids!')
-        pygame.quit()
-    except Exception as e:
-        print(f'Failed to initialize Pygame: {e}')
+    pygame.init()  # Initialize Pygame
+
+    # Set up the display
+    screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+    
+    # Start the game loop
+    clock = pygame.time.Clock()
+    while True:
+        clock.tick(60) 
+        # Event handling to check if the user wants to quit
+        for event in pygame.event.get():
+            if event.type == pygame.QUIT:
+                pygame.quit()
```

Comment:
**Suggestion:** Ensure `pygame.quit()` is called only once to avoid potential issues.

**Existing Code:**
```Python
pygame.quit()
```

**Improved Code:**
```Python
if event.type == pygame.QUIT:
    pygame.quit()
    return  # Exit the game loop if the user closes the window
```

**Details:**
Ensure that `pygame.quit()` is called only once to avoid potential issues when exiting the game loop.